### PR TITLE
Makes "inverseDir" a global proc

### PR DESCRIPTION
**Pull request by utilbird**
## About The Pull Request

title

proc now utilizes switch instead of else if and removes unneeded null return

## Why It's Good For The Game

inversedir could be used elsewhere in the future

## Changelog
:cl:
code: inversedir is now a global proc
/:cl:

### CODE REVIEW

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1602,3 +1602,14 @@ config_setting should be one of the following:
...
+/proc/invertDir(var/input_dir)
+	switch(input_dir)
+		if(NORTH)
+			return SOUTH
+		if(SOUTH)
+			return NORTH
+		if(EAST)
+			return WEST
+		if(WEST)
+			return EAST
```
Comment (francinum) [inline code/__HELPERS/unsorted.dm]:
```suggestion
/proc/invertDir(var/input_dir)
	switch(input_dir)
		if(UP, DOWN)
			CRASH("Can't turn vertical directions!")
		if(1 to 10)
			return turn(input_dir, 180)
		else CRASH("Can't turn invalid directions!")
```

Comment (TheFakeElon) [reply code/__HELPERS/unsorted.dm:1615]: that works, but I'll make it invert up/down too xd

**Comment by utilbird** [comment]
that works, but I'll make it invert up/down too xd

**Comment by francinum** [comment]
shit I was actually rewriting that when you committed.

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -1605,8 +1605,11 @@ config_setting should be one of the following:
...
-		if(UP, DOWN)
-			CRASH("Can't turn vertical directions!")
+		if(UP)
+			return DOWN
+		if(DOWN)
+			return UP
 		if(-INFINITY to 0, 11 to INFINITY)
 			CRASH("Can't turn invalid directions!")
-		else return turn(input_dir, 180)
+			return
```
Comment (francinum) [inline code/__HELPERS/unsorted.dm]: This is unreachable. CRASH causes the proc to die.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**